src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/wiremock/ActivitiesApiMockServer.kt

```diff
@@ -35,7 +35,9 @@ class ActivitiesApiExtension : BeforeAllCallback, AfterAllCallback, BeforeEachCa
 class ActivitiesApiMockServer : WireMockServer(WIREMOCK_PORT) {
   companion object {
     private const val WIREMOCK_PORT = 8086
-    private val objectMapper = jacksonObjectMapper()
+    private val objectMapper = jacksonObjectMapper().apply {
+      findAndRegisterModules()
+    }
```

Comment:
Can now construct stub responses using generated classes instead of hand crafted json strings